tests/files/c_source/vehicle.h

```diff
@@ -256,6 +256,56 @@
 #define VEHICLE_FRAME_ID_RT_SB_ACCEL (0x94a6c22U)
 #define VEHICLE_FRAME_ID_RT_SB_GYRO_RATES (0x94a6d22U)
 
+#define VEHICLE_RT_SB_INS_STATUS_INS_STATUS_VAL_CONVERGED (2U)
+#define VEHICLE_RT_SB_INS_STATUS_INS_STATUS_VAL_INITIALISED (1U)
+#define VEHICLE_RT_SB_INS_STATUS_INS_STATUS_VAL_UNINITIALISED (0U)
```

Comment:
Let's sort the choices in ascending numerical order.

Reply:
Done